spring-integration-debezium/src/main/java/org/springframework/integration/debezium/dsl/Debezium.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2023-2023 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.integration.debezium.dsl;
+
+import java.util.Properties;
+
+import io.debezium.engine.ChangeEvent;
+import io.debezium.engine.DebeziumEngine;
+import io.debezium.engine.DebeziumEngine.Builder;
+import io.debezium.engine.format.KeyValueHeaderChangeEventFormat;
+import io.debezium.engine.format.SerializationFormat;
+
+import org.springframework.util.Assert;
+
+/**
+ * Factory class for Debezium components DSL.
+ *
+ * @author Christian Tzolov
+ *
+ * @since 6.2
+ */
+public final class Debezium {
+
+	/**
+	 * Create an instance of {@link DebeziumMessageProducerSpec} for the provided {@link DebeziumEngine.Builder}.
+	 * @param debeziumEngineBuilder the {@link DebeziumEngine.Builder} to use.
+	 * @return the spec.
+	 */
+	public static DebeziumMessageProducerSpec inboundChannelAdapter(
+			DebeziumEngine.Builder<ChangeEvent<byte[], byte[]>> debeziumEngineBuilder) {
+		return new DebeziumMessageProducerSpec(debeziumEngineBuilder);
+	}
+
+	/**
+	 * Create an instance of {@link DebeziumMessageProducerSpec} for the provided native debezium {@link Properties} and
+	 * JSON serialization formats.
+	 * @param debeziumConfig {@link Properties} with required debezium engine and connector properties.
+	 * @return the spec.
+	 */
+	public static DebeziumMessageProducerSpec inboundChannelAdapter(Properties debeziumConfig) {
+
+		return new DebeziumMessageProducerSpec(builder(
```

Comment:
Why just not delegate to `inboundChannelAdapter(Properties, Class, Class)`?

Reply:
Personally, I condider the telescopic constructors for anti-pattern. For me it is the same as to telescopic public methods in same class.

Reply:
I assume you talk about this this one: https://www.vojtechruzicka.com/avoid-telescoping-constructor-pattern/.

But I don't see a relevance: you build a `Builder` based on a provided props and default formats and then you delegated to the method which is already based on a `Builder` pattern.
How is that telescoping? We don't stack method calls here for extending arguments number.

What do I miss, please?

Good to know that it is an anti-pattern, but sometimes we really have to enforce end-user with our API for this or that choice, where something is so important that it cannot be extracted to a setter.

Reply:
And how about DRY principle?

Reply:
I guess here we should agree to disagree. It is still a bad practice in my book. 
But this is your project and your responsibility so please feel free to rewrite and amend it you think is appropriate before merge.